.github/workflows/m1_tests.yml

```diff
@@ -86,6 +86,13 @@ jobs:
         run: |
           make test_boolean
 
+      # Because we do "illegal" things with the build system which Cargo does not seem to like much
```

Comment:
What is it?
Maybe give more info here

Reply:
was merged in main like that, this is just a backport

Reply:
I don't have the full explanation except it's using a cached build which is not complete see 2 lines below